configs/config.dell_optiplex_9010_sff

```diff
@@ -1,21 +1,24 @@
-CONFIG_LOCALVERSION="Dasharo King's Gambit v0.1"
-CONFIG_USE_OPTION_TABLE=y
+CONFIG_LOCALVERSION="Dasharo Workstation v0.1"
```

Comment:
@miczyg1 I'm not sure if it goes to DMI, but agreed to use `Dasharo (coreboot+SeaBIOS)` for configurations like this

Reply:
The correct DMI is not yet implemented. Can do this quickly today.

Reply:
For verification:
```
Handle 0x0000, DMI type 0, 26 bytes
BIOS Information
	Vendor: 3mdeb Embedded Systems Consulting
	Version: Dasharo (coreboot+SeaBIOS) v0.1.0
	Release Date: 09/24/2021
	ROM Size: 12288 kB
	Characteristics:
		PCI is supported
		PC Card (PCMCIA) is supported
		BIOS is upgradeable
		Selectable boot is supported
		ACPI is supported
		Targeted content distribution is supported
	BIOS Revision: 0.1
	Firmware Revision: 0.0

Handle 0x0001, DMI type 1, 27 bytes
System Information
	Manufacturer: Dell Inc.
	Product Name: OptiPlex 9010
	Version: 1.0
	Serial Number: 123456789
	UUID: Not Settable
	Wake-up Type: Reserved
	SKU Number: Not Specified
	Family: Not Specified

```

Reply:
`BIOS Revision: 0.1` is created now based on the tag on the repo. The tag format must be `dell_optiplex_9010_vx.y.z` so those minor and major revisions are extracted correctly. `dell_optiplex_9010_vx.y.z` format is based on already existing tag ` `dell_optiplex_9010_v0.0.0``

Reply:
LGTM since it is according to https://docs.dasharo.com/dev-proc/source-code-structure/